src/main.tsx

```diff
@@ -1,4 +1,3 @@
-import React from "react";
```

Comment:
is this right?

Reply:
it throws unused import warning so i removed it